src/main/java/io/jenkins/plugins/zscaler/ZscalerScan.java

```diff
@@ -122,7 +122,8 @@ BuildDetails getBuildDetails(Run build, TaskListener listener) throws IOExceptio
     buildDetails.setBuildNumber(String.valueOf(build.getNumber()));
     buildDetails.setBuildTriggeredBy(env.get("USER"));
     buildDetails.addEventDetails("build_url", env.get("BUILD_URL"));
-    buildDetails.addEventDetails("action", build.getDisplayName());
+    buildDetails.addEventDetails("build_number", build.getDisplayName());
```

Comment:
It is not displayName to read build number.

Reply:
getDisplayname is actually giving us the build number like "#24". that was actually storing int action earlier. are you expecting something else in this build number ?